### PR TITLE
Fix sourcecred account migrator

### LINKED ISSUE #468: XP no longer updating on MyMeta
The XP bot is also down, so the only way for players to see their XP is by checking the [https://metafam.github.io/XP/#/accounts](Sourcecred instance) directly.

### PR DESCRIPTION
**Pull request by alalonde**
Upgraded sourcecred to 0.8.4, read cred graph using new sc API

Resolves #468

### CODE REVIEW

File: packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts [hunk 1]
```diff
@@ -71,13 +72,32 @@ const parseAlias = (alias: SCAlias) => {
...
+const ledgerLoadedPromise = manager.reloadLedger();
+
 export const migrateSourceCredAccounts = async (
   _: Request,
   res: Response,
 ): Promise<void> => {
-  const accountsData: SCAccountsData = await (
-    await fetch(ACCOUNTS_FILE)
-  ).json();
+  const ledgerRes: ReloadResult = await ledgerLoadedPromise;
```
Comment (dysbulic) [inline packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts:91]:
```suggestion
  const ledgerRes: ReloadResult = await ledgerLoadedPromise();
```

**Review by dysbulic — COMMENTED** [review]
I'm unfamiliar with the syntax at one point. Can you omit the parens when using await?

File: packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts (hunk 1 above)
Comment (alalonde) [reply packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts:91]: `ledgerLoadedPromise` is of type `Promise<ReloadResult>`.  It is different syntax than usual, but correct.

Usually one would write something like `const ledgerLoadedResult = await manager.reloadLedger()`; Here I'm just assigning the promise to a variable, and awaiting that. 

The benefit here is that `reloadLedger` is called when the backend boots up. We await it to ensure that it is complete before the handler proceeds with execution.

**Review by dysbulic — APPROVED** [review]

File: packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts (hunk 1 above)
Comment (dysbulic) [reply packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts:91]: Gotcha. Very cool. Thanks for adding a trick to my quiver.

**Pull request merged**